slider/replay.py

```diff
@@ -25,8 +25,10 @@ class Action:
 
     Parameters
     ----------
-    offset : timedelta
+    rel_offset : timedelta
```

Comment:
do you think this is ever useful? I can't think of a case where I would want this. It is also very easy to recover by just diffing 2 actions.